### PR TITLE
Improve README: generator, html_class example

### LINKED ISSUE #49: Generator for component
- [x] https://github.com/github/view_component/issues/1067
- [x] Add `bin/rails generate view_component Form::CustomTextField --parent Form::TextField` to README

## Next steps

- `bin/rails generate vcf:component Form::TextField` to generate one component
- `bin/rails generate vcf:components` to generate all components

### PR DESCRIPTION
**Pull request by Spone**
Close #49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,69 +107,67 @@ bin/rails generate vcf:builder AnotherCustomFormBuilder --namespace Forms::Compo
...
 
-You can then customize the behavior of your `Form::TextFieldComponent`:
+You can then customize the behavior of your `Custom::Form::CustomTextFieldComponent`:
 
 ```rb
-# app/components/form/text_field_component.rb
+# app/components/custom/form/text_field_component.rb
+
+class Admin::Form::TextFieldComponent < ViewComponent::Form::TextFieldComponent
+  self.tag_klass = ActionView::Helpers::Tags::TextField
```
Comment (nicolas-brousse) [inline README.md:139]: @Spone I think this line may be removed

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Spone) [reply README.md:139]: If I remove it I get:

```
RuntimeError in Admin::Pages#edit
`self.tag_klass' should be defined in Admin::Form::TextFieldComponent
```

Comment (Spone) [reply README.md:139]: My bad, I wasn't on master.

Comment (nicolas-brousse) [reply README.md]: Weird. https://github.com/pantographe/view_component-form/pull/87 was to fix that :thinking:

**Pull request merged**